Adding small comment on where to find executable when using cargo build (#47)
onomondo/onomondo-softsim-cli: README.md

`softsim next` can be called from manufacturing scripts as needed.

## Build
`cargo build --release`

> When having build the repository, the SoftSIM executeable is located at path: `onomondo-softsim-cli/target/release/softsim`

Reviewer: ```suggestion
After building the SoftSIM executable is located in: `target/release/softsim`
```
